src/components/WishlistCard.tsx

```diff
@@ -45,8 +49,9 @@ export const WishlistCard = ({
   const addToCart = async (id: string, encodedToken: string) => {
     try {
       await postUserCart(id, { authorinzation: encodedToken });
```

Comment:
Ctrl + F this word and replace it everywhere.

Reply:
Yeah just did it